token_server/README.md

```diff
@@ -1,4 +1,37 @@
 # Simple web server to provide EarthEngine tokens to non-whitelisted users
 
-Run with `node --experimental_modules token_server/token_server.js`.
+This is an optional component for the mapping page, that allows any user to view
+it, even if they are not whitelisted with EarthEngine. Only the site maintainer
+should follow the instructions below, and they should only need to be done once.
 
+## Deploy to Google App Engine
+
+* Follow initial steps for Google App Engine deployment: install `gcloud`; when
```

Comment:
maybe helpful to say 'install 'gcloud' (using homebrew or yarn)' or something to that effect to make it abundantly clear we mean the command line tool and the rest of these steps are for the terminal as well

Reply:
Linked to gcloud installation instructions.